src/main/java/org/jabref/logic/exporter/XmpPdfExporter.java

```diff
@@ -0,0 +1,29 @@
+package org.jabref.logic.exporter;
+
+import java.nio.charset.Charset;
+import java.nio.file.Path;
+import java.util.List;
+
+import org.jabref.logic.util.FileType;
+import org.jabref.logic.xmp.XmpPreferences;
+import org.jabref.logic.xmp.XmpUtilWriter;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.entry.BibEntry;
+
+public class XmpPdfExporter extends Exporter {
+
+    private final XmpPreferences xmpPreferences;
+
+    public XmpPdfExporter(XmpPreferences xmpPreferences) {
+        super("pdf", FileType.PDF_XMP.getDescription(), FileType.PDF_XMP);
+        this.xmpPreferences = xmpPreferences;
+    }
+
+    @Override
+    public void export(BibDatabaseContext databaseContext, Path pdfFile, Charset encoding, List<BibEntry> entries) throws Exception {
+        if (pdfFile.toString().endsWith(".pdf")) {
```

Comment:
Is this necessary? I think for an export it should not play a role if my flie is named  `test.asdf` for export, as long as the format is correct?